plugins/psx/src/psx-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.Psx : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "psx";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
Probably not the MIME-type we want, or at least without verification of the file. You can keep it that way for your tests though.